### DISCUSSION TITLE
Authentication

### DISCUSSION THREAD

**Discussion by 1skonty1** [danitseitlin/redis-modules-sdk-ts#171] [Q&A]
Hello. I'm working on a nodejs server. How do I add authorization to this REJSON module in my app. All attempts have failed using this approach.

const client = new ReJSON({
    host: 'localhost',
    port: REDIS_PORT_OUT,
    auth_pass: REDIS_PWD
});
The error showing is:
[ioredis] Unhandled error event: ReplyError: NOAUTH Authentication required.

**Comment by danitseitlin** [comment]
@1skonty1 Your authorization is just a password?

**Reply by danitseitlin** [reply]
@1skonty1
